src/middleware/role-auth.js

```diff
@@ -0,0 +1,21 @@
+const {verifyToken} = require('../helpers/generateToken');
+const accounts = require('../models/accounts');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const accounts = require("../models/accounts");
```